lib/ain-evm/src/evm.rs

```diff
@@ -143,6 +152,21 @@ impl EVMServices {
 
         let mut executor = AinExecutor::new(&mut backend);
 
+        if current_block_number == U256::zero() {
```

Comment:
some note here.. it won't work until we remove the genesis restriction on regtest only and dependence at `ethStartState` flag

Reply:
Would it work if we do this on block 1?

Reply:
yea block 1 will work.. can add a note above the line? 
```
TODO(): change back to `U256::zero()` after https://github.com/DeFiCh/ain/issues/2216
```

Reply:
Ignore the thread.. some memory conflicts..
the block 0 exists before feature/evm enable
It's deployed when compiling the defid